app/src/components/account/SignInModal.vue

```diff
@@ -35,41 +45,11 @@ export default {
       await this.$store.dispatch(`sendUserMagicLink`, {
         user: { email: this.email },
       })
-      this.$router.push({ name: `magic-link-sent-modal` })
+      this.$router.push({ name: `magic-link-sent` })
     },
     close() {
       this.$router.go(`-1`)
     },
   },
 }
 </script>
-<style scoped>
-.sign-in-buttons {
-  display: flex;
-  justify-content: space-evenly;
-  margin-top: 1.5em;
-}
-
-.sign-in-buttons button {
-  width: 100%;
-  height: 4rem;
-}
-
-.sign-in-buttons button:first-child {
-  margin-right: 0.8rem;
-}
-
-.sign-in-buttons button:nth-child(2) {
-  margin-left: 0.8rem;
-}
-
-#email {
-  height: 3.5rem !important;
-}
-
-@media screen and (min-width: 667px) {
-  .account {
-    min-width: 460px;
-  }
-}
-</style>
```

Comment:
:eyes:

Reply:
Personally I think it looked better with the styled that Mario first implemented following Charlie's designs:

![image](https://user-images.githubusercontent.com/40721795/86762809-2db0e480-c047-11ea-8468-0b386870b521.png)

VS

![image](https://user-images.githubusercontent.com/40721795/86762611-0d812580-c047-11ea-9c85-3601854b3a27.png)


Looks more stylish to me at least with the big input and big button. More pleasant to the eye

Reply:
we will get this over to charlie's design eventually. i don't want to have random input and button styles in one place.